configs/mmdet3d/monocular-detection/monocular-detection_static.py

```diff
@@ -0,0 +1,14 @@
+_base_ = ['../../_base_/onnx_config.py']
+
+onnx_config = dict(
+    input_names=['img', 'cam2img', 'cam2img_inverse'],
+    output_names=['bboxes', 'scores', 'labels', 'dir_scores', 'attrs'],
+    input_shape=None,
+)
+codebase_config = dict(
+    type='mmdet3d',
+    task='MonocularDetection',
+    model_type='end2end',
+    ann_file='tests/test_codebase/test_mmdet3d/data/nuscenes/n015-2018-07-24'
```

Comment:
It is not a good idea to place ann_file in deploy config. Any better idea?
@RunningLeon @tpoisonooo @lvhan028 @hanrui1sensetime @AllentDan

Reply:
Can we get `cam_intrinsic` from `model_cfg`?

Reply:
Our group discussed this and I thought of many ways to implement cam_intrinsic, but I failed to implement it beautifully without hurting mmdet3d and mmdeploy. In mmdet3d, we get `cam_intrinsic` from ann_file which is additional parameters in mmdet3d api, generally feeding on command line.

Reply:
Well, I think we can accept it temporarily. Let's figure out a better solution in next release.